.gitignore

```diff
@@ -16,7 +16,7 @@ __pycache__/
 /.coverage
 /artiq/test/lit/*/Output/
 /artiq/binaries
-/artiq/firmware/target/
+/artiq/firmware/**/target/
```

Comment:
Shouldn't be part of this PR (if it should be there at all).

Reply:
Should I open a new PR for this line?

Reply:
If you need it. But I'm not sure what problem this fixes.